medicai/models/unet/unet_decoder.py

```
@@ -61,26 +75,56 @@ def apply(x, skip=None):
                 strides=2,
                 padding="same",
+                name=f"{stage_prefix}_upconv_{dim_str}",
             )(x)
+            if use_batchnorm:
+                x = layers.BatchNormalization(axis=-1, name=f"{stage_prefix}_bn_{dim_str}")(x)
+            x = layers.Activation("relu", name=f"{stage_prefix}_relu")(x)
         else:
             x = get_reshaping_layer(spatial_dims, layer_type="upsampling", size=2)(x)
 
         if skip is not None:
+            # Resize skip if spatial shapes don’t match
```

Ruff (0.13.3)

Comment contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF003)